### DISCUSSION TITLE
Playlist memory

### DISCUSSION THREAD

**Discussion by kc3ol** [AzuraCast/AzuraCast#6612] [General]
If I restart my server/Azuracst it starts the playlist from the beginning.   It would be much better if the playlist started from where it was previously.

**Comment by ChandigarIT** [comment]
Similar request regarding an interrupted track, if a recorded show is interrupted by a higher weight playlist, when the 2nd playlist has ended, the first show comes back on, at the start, which is not good if it was nearly at the end of a 2-hour show.
